CSharp/ArticleProjects/EfficientListQueries/EfficientListQueries.csproj

```diff
@@ -36,8 +36,8 @@
       <HintPath>packages\Hyak.Common.1.0.2\lib\net45\Hyak.Common.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Batch, Version=2.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>packages\Azure.Batch.2.0.2\lib\net45\Microsoft.Azure.Batch.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Batch, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>packages\Azure.Batch.3.0.0\lib\net45\Microsoft.Azure.Batch.dll</HintPath>

```

Comment:
Fred says it's ok to merge. I'll get that done later today.
________________________________
From: jasper-schneider<mailto:notifications@github.com>
Sent: ‎12/‎23/‎2015 3:06 PM
To: Azure/azure-batch-samples<mailto:azure-batch-samples@noreply.github.com>
Subject: Re: [azure-batch-samples] Using Azure Batch SDK 3.0 and update batch explorer for new feature (MPI and online/offline) (#94)


In CSharp/ArticleProjects/EfficientListQueries/EfficientListQueries.csproj<https://github.com/Azure/azure-batch-samples/pull/94#discussion_r48386042>:

> @@ -36,8 +36,8 @@
>        <HintPath>packages\Hyak.Common.1.0.2\lib\net45\Hyak.Common.dll</HintPath>
>        <Private>True</Private>
>      </Reference>
> -    <Reference Include="Microsoft.Azure.Batch, Version=2.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
> -      <HintPath>packages\Azure.Batch.2.0.2\lib\net45\Microsoft.Azure.Batch.dll</HintPath>
> +    <Reference Include="Microsoft.Azure.Batch, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
> +      <HintPath>packages\Azure.Batch.3.0.0\lib\net45\Microsoft.Azure.Batch.dll</HintPath>


Charlie already made these changes in the sample projects, he was just dealing with a Travis error before merging: #92<https://github.com/Azure/azure-batch-samples/pull/92>

You might want to just wait for his to go through, then resubmit with only the Batch Explorer changes.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-batch-samples/pull/94/files#r48386042>.